raft/src/main/java/org/apache/kafka/raft/QuorumStateStore.java

```diff
@@ -18,20 +18,28 @@
 
 import java.io.IOException;
 
-public interface ElectionStore {
+/**
+ *  Maintain the save and retrieval of quorum state information, so far only supports
+ *  read and write of election states.
+ */
+public interface QuorumStateStore {
+
+    int UNDEFINED_EPOCH = -1;
```

Comment:
Hmm.. Why do we need this? I think we should always have an epoch. Even if there is no previous state, the epoch should be 0.

Reply:
Sg, will remove